rooms/windyTylne.cpp

```diff
@@ -1,24 +1,212 @@
+#include <cstdlib>
 #include <iostream>
+#include <string>
 
 #include "../main.hpp"
 
-Room windyTylne(__attribute__((unused)) GameData &gameData) {
-  std::cout << "Jesteś w windach tylnych" << std::endl
-            << "Czy chcesz iść do biblioteki?" << std::endl;
-  if (yes()) {
-    return BIBLIOTEKA;
-  }
-  std::cout << "Czy chcesz iść do garażu?" << std::endl;
-  if (yes()) {
-    return GARAZ;
-  }
-  std::cout << "Czy chcesz iść do windy na piętrze?" << std::endl;
-  if (yes()) {
-    return WINDA_PIETRO;
+const int8_t minigame_row = 3;
+const int8_t minigame_col = 3;
+
+void start(GameData& gameData) {
+  std::cout << "Przechodzisz do tylnej strony budynku i widzisz "
+            << (gameData.winda_repaired
+                    ? "naprawioną przez siebę windę "
+                    : "windę, która sprawia wrażenie niedziałającej")
+            << std::endl;
+}
+/*
+void draw_minigame(char k[][minigame_col+2], short off[minigame_col]) {
+    std::cout << "Not yet implemented" << std::endl;
+}
+*/
+void input_prompt(std::string& s) {
```

Comment:
Write a proper menu interface, not... well... this thing

Reply:
It's temporary, proper menu interface is being written for whole project. When it'll be implemented&tested I'll change that